docs/KeyDecisions.md

```diff
@@ -36,7 +36,7 @@ Note that negation of type integer is not fully defined yet as we do not have a
 
 Given the above, should we expect nulls? If null is considered a _value_ then no would be a reasonable answer, but it can equally be considered the absence of a value. 
 
-## What do temporal constraints mean when datetimes are partially specified?
+## What do datetime constraints mean when datetimes are partially specified?
 
 ```javascript
 { "field": "creationDate", "is": "after", "value": "2015-01-01" }
```

Comment:
Unrelated note: Need to double check that dates can be specified in this format, I thought they always needed to be specified in full ISO-8601 format and within a 'date object.